shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/java/org/apache/shardingsphere/sql/parser/sqlserver/visitor/impl/SQLServerDMLVisitor.java

```diff
@@ -469,4 +469,9 @@ public ASTNode visitGroupByClause(final GroupByClauseContext ctx) {
         }
         return new GroupBySegment(ctx.getStart().getStartIndex(), ctx.getStop().getStopIndex(), items);
     }
+    
+    @Override
+    public ASTNode visitSubquery(final SQLServerStatementParser.SubqueryContext ctx) {
+        return visit(ctx.unionClause());
+    }
```

Comment:
Do you think `SQL92DMLVisitor` need this function as well?

Reply:
Ok.